### PR TITLE
Feature meal branch

### PR DESCRIPTION
**Pull request by bulic**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,15 +1,343 @@
...
+  }
+}
+
+@media only screen and (max-width: 768px) {
+  .about-you {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 20%;
```
Comment (BraneOpacic21) [inline style.css]: Don't use percent for margin, especially large ones as this. 

 Also, if you are trying to 'push' an element from another - good advice here would be to use the `margin-bottom` on the upper element - follow that implementation everywhere throughout the app

File: style.css [hunk 2]
```diff
@@ -1,15 +1,343 @@
...
+  }
+
+  .meals {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-top: 9%;
```
Comment (BraneOpacic21) [inline style.css]: Same as above

File: style.css [hunk 3]
```diff
@@ -1,15 +1,343 @@
...
+    justify-content: center;
+    align-items: center;
+  }
+  .city-title {
+    color: #0a0a0a;
+    font-size: 2.2em;
+    text-align: center;
+    font-weight: bold;
+    line-height: 1em;
+    padding-left: 0%;
```
Comment (BraneOpacic21) [inline style.css]: Could this be written better?

File: style.css [hunk 4]
```diff
@@ -1,15 +1,343 @@
...
+  padding: 0.8em;
+  width: 45%;
+  text-transform: uppercase;
+}
+
+footer {
+  margin: 5em 0em;
+  display: flex;
+  flex-direction: column;
+  padding: 0em 28%;
```
Comment (BraneOpacic21) [inline style.css]: Avoid using percentages for padding and margin if you may. We will talk about this subject in the meeting to discuss it further

File: style.css [hunk 5]
```diff
@@ -1,15 +1,343 @@
...
+  justify-content: center;
+  margin-left: -1em;
+}
+
+.city {
+  display: flex;
+  flex-direction: column;
+  width: 47%;
+  justify-content: center;
+}
```
Comment (BraneOpacic21) [inline style.css]: This whole section should probably be done differently. 

`margin-left: -1em;` seems like a hack to move an element just slightly, and using 53% and 47% is not something that's considered a best practice. 

We should discuss this (in our meeting) and find the best solution, avoiding confusion. 😺

**Review by aberonja — COMMENTED** [review]
Resolve conflicts first 👍

File: style.css [hunk 6]
```diff
@@ -82,76 +87,241 @@ a:hover {
...
+
+.vege-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.rotateimg180 {
```
Comment (BraneOpacic21) [inline style.css]: Let's keep the _dash_ class naming if we're already using it. Also, suggestion: `img-reverse` maybe? Or something similar 😄

File: style.css [hunk 7]
```diff
@@ -82,76 +87,241 @@ a:hover {
...
+.meal-block {
+  display: flex;
+  margin-bottom: 2em;
+}
+
+.meal-image {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
```
Comment (BraneOpacic21) [inline style.css:210]: Just a general suggestion here: In the future, you could use something like `flex-center-center` class, which will have the corresponding CSS rules, so you don't have to repeat yourself (DRY principle) everywhere you plan to center an element (or else).

P.S.
Don't need to change anything here, just for future reference 😄 

Keep in mind that the CSS frameworks (e.g. Bootstrap) use something like I mentioned above 🥇 

Adding @dsekulovic to read this hint, as well.

File: style.css [hunk 8]
```diff
@@ -82,76 +87,241 @@ a:hover {
...
+}
+
+.section-three {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  align-items: center;
+  background-color: #ffffff;
+  width: 100%;
+  padding: 0em 1em 3em 1em;
```
Comment (BraneOpacic21) [inline style.css]: Nitpicking, but this could be written shorter. (We have enough time, so let's learn shorthands also 😄 )

File: style.css [hunk 9]
```diff
@@ -170,29 +340,88 @@ img {
...
+  }
+
+  .meal-block {
+    flex-direction: column;
+    margin: 0em 0.3em;
+  }
+
+  .section-three {
+    flex-direction: row;
+    padding: 0em 5em 3em 0em;
```
Comment (BraneOpacic21) [inline style.css:392]: shorthand

**Review by BraneOpacic21 — CHANGES_REQUESTED** [review]
Nice work, in general 👍 

Keep in mind that it's better to call your CSS classes with a bit more descriptive names.

For example, `section-one` and `section-two` could be (something like) `our-work` and `contact-us`.

Suggestion, for future reference. It would be easier to understand from the code which part of the app is the section referring to 😃

File: style.css (hunk 9 above)
Comment (BraneOpacic21) [reply style.css:392]: Is this resolved? 🛩️

File: index.html [hunk 10]
```diff
@@ -34,12 +34,12 @@ <h1 class="big-title">
...
             <span class="red-span">Hungry</span> delicious.
           </h1>
-          <p>
+          <p class="left-align">
             In order to build a successful brand, every food selector business -
             whether retail, wholesale, needs to make a connection with
             customers.
           </p>
-          <div class="input-button-field">
+          <div class="input-button-field" >
```
Comment (aberonja) [inline index.html]: Use some kind of linter to prevent empty spaces like you have it here after the class declaration.

File: style.css (hunk 9 above)
Comment (bulic) [reply style.css:392]: If we have top and left same values in padding I don't know how  to create shorthand?

File: index.html [hunk 11]
```diff
@@ -51,18 +51,154 @@ <h1 class="big-title">
           </div>
         </div>
         <div class="part-two">
-          <img src="big-meal.png" alt="big meal" />
+          <img src="big-meal.png" alt="big meal" class='big-meal-photo' />
```
Comment (aberonja) [inline index.html]: This (**class='big-meal-photo'**) will also be fixed when you implement linter.

File: index.html [hunk 12]
```diff
@@ -51,18 +51,154 @@ <h1 class="big-title">
...
         </div>
       </section>
-      <section>
-        <h2>Second section</h2>
-        <div>Add text and images</div>
+      <section class="section-two">
+          <div class="head-meal-section">
+            <h4 class="head-meal-title">
+              Don't just pick a restaurant </br>
```
Comment (aberonja) [inline index.html]: Please, whenever you can try not to use break.

File: index.html [hunk 13]
```diff
@@ -51,18 +51,154 @@ <h1 class="big-title">
...
+              <span class="red-span">
+                pick your meal
+              </span>
+            </h4>
+          </div>
+          <p class="under-head-meal-section">
```
Comment (aberonja) [inline index.html]: Please, read some posts about class naming conventions, it'll help you a bit.

File: index.html [hunk 14]
```diff
@@ -51,18 +51,154 @@ <h1 class="big-title">
...
+            Gustar offers you the perfect meal options based on your<br />
+            favorite food, places to eat, and budget.
+          </p>
+          <div class="about-you">
+            <div class="vege-card">
+              <img src="carrot.png" width = "50em" height= "50em"  />
```
Comment (aberonja) [inline index.html]: lint

File: index.html [hunk 15]
```diff
@@ -51,18 +51,154 @@ <h1 class="big-title">
...
+                <p class="vege-text">
+                  Gustar offers you the perfect meal options based on your
+                  favorite food, places to eat, and budget
+                </p>
+              </div>
+            </div>
+            <div class="vege-card">
+              <img
+                src="food-broccoli.png"
+                width = "50em" height= "50em"
```
Comment (aberonja) [inline index.html]: lint (I won't comment it anymore, linter will fix it)

File: style.css [hunk 16]
```diff
@@ -16,7 +16,6 @@ body {
   background-color: #ffffff;
   font-size: 16px;
 }
```
Comment (aberonja) [inline style.css]: Why did you remove \n?

File: style.css (hunk 9 above)
Comment (BraneOpacic21) [reply style.css:392]: My bad, you are right - I read it incorrectly 😄

Comment (bulic) [reply style.css:392]: ;)

File: style.css [hunk 17]
```diff
@@ -170,29 +340,88 @@ img {
...
   }
+  .section-three {
+    width: 80%;
+  }
 }
 
 @media only screen and (min-width: 1368px) {
   .main-container {
     width: 60%;
   }
```
Comment (aberonja) [inline style.css:425]: \n

File: style.css [hunk 18]
```diff
@@ -170,29 +340,88 @@ img {
...
+
+  .input-padding {
+    padding: 0em 8em;
+  }
 }
 
 @media only screen and (min-width: 992px) {
   .main-container {
     width: 80%;
   }
```
Comment (aberonja) [inline style.css:415]: \n

**Review by aberonja — CHANGES_REQUESTED** [review]

File: index.html (hunk 14 above)
Comment (bulic) [reply index.html]: I use linter and on only img element it set with space on the end like this.

File: index.html (hunk 15 above)
Comment (bulic) [reply index.html]: I use linter and only on img element it set with space on the end like this.

File: index.html (hunk 14 above)
Comment (aberonja) [reply index.html]: **width = "50em"** This is the problem, I saw it throughout the PR. Spacing at the end is fine except it adds two spaces which is not necessary.

File: .vscode/settings.json [hunk 19]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (aberonja) [inline .vscode/settings.json:1]: I assume that this file is not necessary, @BraneOpacic21?

Comment (BraneOpacic21) [reply .vscode/settings.json:1]: I would agree, considering it only has an empty object literal inside 😸

Comment (aberonja) [reply .vscode/settings.json:1]: @bulic please remove it You can place it in .gitignore file.

**Review by aberonja — APPROVED** [review]

**Comment by aberonja** [comment]
💃
